.ci/openjdk-projects-to-test-on.properties

```diff
@@ -0,0 +1,11 @@
+# suppress inspection "UnusedProperty" for whole file
+# suppress inspection "WrongPropertyKeyValueDelimiter" for whole file
```

Comment:
please change extension of file to ".config" to skip this file from any tool that try to validate it.

I do not remember why we named it as ".properties", probably very first version of file was based on properties.

Reply:
Done.